include/nuttx/video/fb.h

```diff
@@ -232,258 +232,296 @@
+
+#define FB_ACCEL_PUV3_UNIGFX         0xa0  /* PKUnity-v3 Unigfx              */
+
+#define FB_NONSTD_HAM                1     /* Hold-And-Modify (HAM)          */
+#define FB_NONSTD_REV_PIX_IN_B       2     /* Order of pixels in each byte   */
+                                           /* is reversed                    */
+
+#define FB_ACTIVATE_NOW              0     /* Set valus immediately (or vbl) */
+#define FB_ACTIVATE_NXTOPEN          1     /* Activate on next open          */
+#define FB_ACTIVATE_TEST             2     /* Don't set, round up impossible */
+#define FB_ACTIVATE_MASK             15    /* values                         */
+#define FB_ACTIVATE_VBL              16    /* Activate values on next vbl    */
+#define FB_CHANGE_CMAP_VBL           32    /* Change colormap on vbl         */
+#define FB_ACTIVATE_ALL              64    /* Change all VCs on this fb      */
+#define FB_ACTIVATE_FORCE            128   /* Force apply even if no change  */
+#define FB_ACTIVATE_INV_MODE         256   /* Invalidate videomode           */
+
+#define FB_ACCELF_TEXT               1     /* (OBSOLETE) See fb_info.flags   */
+                                           /* and vc_mode                    */
+
+#define FB_SYNC_HOR_HIGH_ACT         1     /* Horizontal sync high active    */
+#define FB_SYNC_VERT_HIGH_ACT        2     /* Vertical sync high active      */
+#define FB_SYNC_EXT                  4     /* External sync                  */
+#define FB_SYNC_COMP_HIGH_ACT        8     /* Composite sync high active     */
+#define FB_SYNC_BROADCAST            16    /* Broadcast video timings        */
+                                           /* vtotal = 144d/288n/576i -  PAL */
+                                           /* vtotal = 121d/242n/484i - NTSC */
+#define FB_SYNC_ON_GREEN             32    /* Sync on green                  */
+
+#define FB_VMODE_NONINTERLACED       0     /* Non interlaced */
+#define FB_VMODE_INTERLACED          1     /* Interlaced */
+#define FB_VMODE_DOUBLE              2     /* Double scan */
+#define FB_VMODE_ODD_FLD_FIRST       4     /* Interlaced: top line first */
 #define FB_VMODE_MASK                255
 
-#define FB_VMODE_YWRAP               256    /* Ywrap instead of panning */
-#define FB_VMODE_SMOOTH_XPAN         512    /* Smooth xpan possible */
-                                            /* (internally used) */
-#define FB_VMODE_CONUPDATE           512    /* Don't update x/yoffset */
+#define FB_VMODE_YWRAP               256   /* Ywrap instead of panning */
+#define FB_VMODE_SMOOTH_XPAN         512   /* Smooth xpan possible */
+                                           /* (internally used) */
+#define FB_VMODE_CONUPDATE           512   /* Don't update x/yoffset */
 
 #define FB_ROTATE_UR                 0
 #define FB_ROTATE_CW                 1
 #define FB_ROTATE_UD                 2
 #define FB_ROTATE_CCW                3
 
+#define ARGBTO32(a,r,g,b) \
```

Comment:
why not move to include/nuttx/video/rgbcolors.h

Reply:
I don't think it's used any more, and I forgot to remove it. There is no dependency on rgbcolours any more so I will check/fix.

Reply:
Yes, I have moved (back) to use rgbcolors.h included in fb.h only when `#ifdef CONFIG_VIDEO_FB_SPLASHSCREEN`